<ui version="4.0" >
 <author>Matthew Livingstone</author>
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="AddWidget" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="RemoveWidget" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget" />
   </item>
   <item>
    <widget class="QLineEdit" name="ToServer" />
   </item>
   <item>
    <widget class="QLineEdit" name="FromServer" />
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pqUI.qrc" />
 </resources>
 <connections/>
</ui>
